src/server/handler.js

```diff
@@ -1,107 +1,122 @@
+const getData = require('../queries/getData');
+const postData = require('../queries/postData');
+const qs = require('querystring');
 
-const homeHandler = (request, response) => {
-	const filePath = path.join(__dirname, "..", "..", "public", "index.html");
+const cookie = require('cookie');
+const jwt = require('jsonwebtoken');
+
+const { parse } = require('url');
+
+const homeHandler = (page, request, response) => {
+	let filePath = path.join(__dirname, '..', '..', 'public', `${page}.html`);
+	if (page === 'profile') {
+		console.log('cookies: ', request.headers.cookie);
+		console.log('login status from cookie', cookie.parse(request.headers.cookie).loggedIn);
+		if (cookie.parse(request.headers.cookie).loggedIn === 'true') {
+			filePath = path.join(__dirname, '..', '..', 'public', `${page}.html`);
+		} else {
+			filePath = path.join(__dirname, '..', '..', 'public', 'index.html');
+		}
+	}
 	fs.readFile(filePath, (error, file) => {
 		if (error) {
 			console.log(error);
 			response.writeHead(500, {
-				"Content-Type": "text/html"
+				'Content-Type': 'text/html'
 			});
-			response.end("<h1>Server Error</h1>");
+			response.end('<h1>Server Error</h1>');
 		} else {
 			response.writeHead(200, {
-				"Content-Type": "text/html"
+				'Content-Type': 'text/html'
 			});
 			response.end(file);
 		}
 	});
 };
 
 const publicHandler = (request, response, endpoint) => {
-	const extension = endpoint.split(".")[1];
+	const extension = endpoint.split('.')[1];
 	const extensionType = {
-		html: "text/html",
-		css: "text/css",
-		js: "application/javascript",
-		png: "image/png",
-		jpg: "image/jpg"
+		html: 'text/html',
+		css: 'text/css',
+		js: 'application/javascript',
+		png: 'image/png',
+		jpg: 'image/jpg'
 	};
 
-	const filePath = path.join(__dirname, "..", "..", endpoint);
+	const filePath = path.join(__dirname, '..', '..', endpoint);
 
 	fs.readFile(filePath, (error, file) => {
 		if (error) {
 			console.log(error);
 			response.writeHead(500, {
-				"Content-Type": "text/html"
+				'Content-Type': 'text/html'
 			});
-			response.end("<h1>Server Error</h1>");
+			response.end('<h1>Server Error</h1>');
 		} else {
 			response.writeHead(200, {
-				"Content-Type": extensionType[extension]
+				'Content-Type': extensionType[extension]
 			});
 			response.end(file);
 		}
 	});
 };
 
 const cuisineHandler = (request, response) => {
-	const type = request.url.split("=")[1];
+	const type = request.url.split('=')[1];
+	console.log('type', type);
 	getData(type, (error, result) => {
 		if (error) {
 			return errorHandler(request, response);
 		}
-		console.log("result", result);
+		console.log('result', result);
 
 		let dynamicData = JSON.stringify(result);
 		response.writeHead(200, {
-			"Content-Type": "application/json"
+			'Content-Type': 'application/json'
 		});
 		response.end(dynamicData);
 	});
 };
 
 const addRestaurantHandler = (req, res) => {
-	let body = "";
-	req.on("data", chunk => {
+	let body = '';
```

Comment:
Okay, you need to sort out your linting to make it consistent for everyone (same config for everyone). Either pick single or double quotes.